src/react/site/home/index.tsx

```diff
@@ -123,10 +142,16 @@ function Home({ onLoad }: { onLoad: () => void }) {
                   ))}
             </div>
           </div>
-          <Threads.Root className="mt-3">
-            {lastThreads?.map((thread) => {
-              return <Threads.Thread key={thread.url} {...thread} />;
-            })}
+          <Threads.Root className="mt-3 h-screen">
```

Comment:
A different approach would be passing the threads as prop to this component and implementing the logic for the "loading threads" there if the list is empty or undefined.

Reply:
Honestly, it's a very good idea. 

The only issue, is that if we go that way, then to me feels weird that it also doesn't take care of rendering the normal threads, that's my only con.